src/data/contributors/cormackrum.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
You should not remove other's data, please revert

Reply:
Hello @Abhishek-kumar09 
Actually I have not removed any other's data, I have just updated my own.
I had that username for a while then i updated to this one.

Reply:
#129 See this one.